tests/suites/os/tests/connectivity/docker-compose.yml

```diff
@@ -5,4 +5,4 @@ services:
     network_mode: host
     restart: on-failure
     command: -verbose -listen :8123
-    user: "995" # run as redsocks user to avoid loop
+    user: redsocks # run as redsocks user to avoid loop
```

Comment:
I just remembered that this will run as the `redsocks` user in the container namespace, it won't necessarily align to the `redsocks` user in the host namespace if that was the original intent of using `995`.